### ISSUE TITLE
Translating the docs

### ISSUE THREAD

**Issue by marioortizmanero** [vidify/vidify#73]
Translated documentation would really help some users. I can translate it to Spanish myself, but if there are any volunteers, I'd love it :)

What needs to be translated:
* README
* Wiki
* In-app content (no support for translations currently)

**Comment by AndrewAmmerlaan** [comment]
I could do Dutch (and I could *try* German, my German is a bit rusty though so maybe I should just stick with Dutch). How would this work? I saw something that looked like 'a mapping of text to translated text' in source code once, but to be honest I don't have any idea how that worked. Or do we just add new translated files (e.g. README_NL_nl.md, etc)?

**Comment by marioortizmanero** [comment]
Oh thanks a bunch Andrew!

I think the easiest way to do it is by creating a new README for every language, as you suggested. Something like `README.es-ES.md`. Then the main english README would have links to these translations.

About the wiki, are you able to create a new article right now? (as in, do you need a special role to create one?) I think what's best is creating new translated article and fixing the navbar to include the languages.

**Comment by AndrewAmmerlaan** [comment]
> About the wiki, are you able to create a new article right now? (as in, do you need a special role to create one?) I think what's best is creating new translated article and fixing the navbar to include the languages.

I'm not able to edit the wiki on github itself (or at least I can't find the button to do that), but I can clone https://github.com/vidify/vidify.wiki.git , translate those files locally and then re-upload it here?

**Comment by marioortizmanero** [comment]
I didn't even know cloning wikis was a thing lol. Sure, that seems like the best way to translate it.

Now that I'm working on the [website](https://github.com/vidify/vidify-website), it might even be better to use it instead. It uses markdown as well so it'd be as easy as a copy-paste.

**Comment by AndrewAmmerlaan** [comment]
[vidify.wiki.tar.gz](https://github.com/vidify/vidify/files/4420721/vidify.wiki.tar.gz)

Here's a tarball containing the translated versions of the files currently on the wiki.

I'll do the README later today

**Comment by marioortizmanero** [comment]
Wow that was fast! Thanks a bunch for the contribution :)

**Comment by AndrewAmmerlaan** [comment]
The README-nl_NL can be found in the dutch-translation branch of my fork: https://github.com/AndrewAmmerlaan/vidify/tree/dutch-translation
I figured I'd upload it to my GitHub fork to make it easier to track new changes to the original README and find exactly what needs translating in the future.

> Wow that was fast! 
COVID-19 is messing with my regular work, so I was very happy to find something to do :D

**Comment by marioortizmanero** [comment]
Hahaha I'm working more on Vidify because of COVID-19, too. Are you going to make a PR, then?

**Comment by AndrewAmmerlaan** [comment]
> Are you going to make a PR, then?

Ooh oops, I forgot to do that xD

Here it is: https://github.com/vidify/vidify/pull/81

**Comment by marioortizmanero** [comment]
Update: I've translated the official site to [Spanish](vidify.org/es), and the wiki to [Dutch](vidify.org/nl/wiki). I was thinking of only offering translations outside of GitHub. This repo's content would only be in English, which isn't a problem for most developers IMO. Your README translation was helpful because I've distributed its contents throughout the site, but I'm not sure if I should remove it. What do you think?

**Comment by AndrewAmmerlaan** [comment]
> Update: I've translated the official site to Spanish, and the wiki to Dutch. I was thinking of only offering translations outside of GitHub. This repo's content would only be in English, which isn't a problem for most developers IMO, and I'm not sure if maintaining the README translated is worth it.
>
>Your README translation was helpful because I've distributed its contents throughout the site, but I'm not sure if I should remove it. What do you think?

Yeah I guess having translations in the repo is not really necessary. I suppose the README_nl_NL can go.

I'm glad it was useful :) I could look into translating the rest of the website some time this week. It shouldn't be that much work as it is not that much text. Website looks really nice by the way!

**Comment by marioortizmanero** [comment]
Great! I'll remove it soon, then. Thanks for your help :smile:
